test/strong_mode_test.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library analyzer_cli.test.strong_mode;
+
+import 'dart:io';
+import 'dart:mirrors';
+import 'package:path/path.dart' as path;
+import 'package:unittest/unittest.dart';
+
+/// End-to-end test for --strong checking.
```

Comment:
:+1:

Nope.  You didn't overlook anything.  Integration testing happens elsewhere.  We've talked about adding some here too (and I share your nervousness) so I think this is just great!